Oracle.NoSQL.SDK/src/Oracle.NoSQL.SDK.csproj

```diff
@@ -9,14 +9,15 @@
     <Copyright>Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.</Copyright>
     <Description>Oracle NoSQL .NET SDK</Description>
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+	<PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageProjectUrl>https://github.com/oracle/nosql-dotnet-sdk</PackageProjectUrl>
     <RepositoryUrl>https://github.com/oracle/nosql-dotnet-sdk</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageTags>oracle cloud oracle-cloud nosql database</PackageTags>
     <PackageReleaseNotes>Initial release.
     </PackageReleaseNotes>
     <PackageIcon>Icon.png</PackageIcon>
-    <Version>5.1.3</Version>
+    <Version>5.1.4</Version>
```

Comment:
We have bumped all of the other SDKs to "5.3.x" for V3

Reply:
Ok, so i guess it should be 5.3.1 then? I ll change it.